libfwupdplugin/fu-hid-device.h

```diff
@@ -16,11 +16,7 @@ G_DECLARE_DERIVABLE_TYPE (FuHidDevice, fu_hid_device, FU, HID_DEVICE, FuUsbDevic
 struct _FuHidDeviceClass
 {
 	FuUsbDeviceClass parent_class;
-	gboolean	 (*open)		(FuHidDevice		*device,
-						 GError			**error);
-	gboolean	 (*close)		(FuHidDevice		*device,
-						 GError			**error);
-	gpointer	__reserved[29];
+	gpointer	__reserved[31];
```

Comment:
ABI change no?

Reply:
Yes, absolutely. I pondered leaving the code so that we called `->close()` like before for any plugins in development (maybe with `g_warning()`) and also pondered just leaving the vfuncs with an `/* unused */` comment. Ideas?

Reply:
I added them back with g_warning() and added them to README.txt for the next time we bump the API. You're completely correct, again :)